Src/World/Timing.h

```diff
@@ -5,38 +5,46 @@
 
 class Timing {
     private:
-        double timeStep;
-        double accumulatedSeconds;
-
-        double secondToLastFixedTime;
-        double lastFixedTime;
-
-        // Total time since the object's initialization.
         std::chrono::high_resolution_clock::time_point initialTime;
-        // Previous time when the last call to getElapsedSeconds() was made.
         std::chrono::high_resolution_clock::time_point prevTime;
 
-    public:
-        Timing(int tickrate);
-
-        double getTimeStep() const;
-        void addSecondsToAccumulator(double secondsPassed);
-
-        // Returns whether enough time is left on the accumulator for another tick.
-        bool tickReady() const;
+        // Tick timers (Milliseconds)
```

Comment:
This says milliseconds, yet the values you're working with in nanoseconds.

Reply:
tickStep is nanoseconds, and sinceLastTick is in fractions of a second, which has been generally described as milliseconds.
I have added a few more comments describing each piece.